src/CONST.js

```diff
@@ -24,6 +24,16 @@ const CONST = {
             POPOVER: 'popover',
             RIGHT_DOCKED: 'right_docked',
         },
+        ANCHOR_ORIGIN_VERTICAL: {
```

Comment:
This positioning schema is inspired by material-ui's popovers https://material-ui.com/components/popover/